eth_client/test/eth_client_test.exs

```diff
@@ -1,14 +1,12 @@
 defmodule EthClientTest do
   use ExUnit.Case
   doctest EthClient
-  alias EthClient.Account
-  alias EthClient.Context
 
   @bin "../contracts/src/bin/Storage.bin"
   @abi "../contracts/src/bin/Storage.abi"
 
-  setup %{bin: bin, abi: abi} do
-    contract = EthClient.deploy(bin, abi)
+  setup_all do
+    contract = EthClient.deploy(@bin, @abi)
```

Comment:
With this change, you can now remove the tags from the tests (lines 39, 64, etc)